tests/llm/test_holmes_checks.py

```
@@ -0,0 +1,316 @@
+        try:
+            if "eval_span" in locals():
+                eval_span.log(
+                    input=str(test_case.checks),
+                    output=str(actual_results) if actual_results else str(e),
+                    expected=str(test_case.expected_results),
+                    dataset_record_id=test_case.id,
+                    scores={},
+                    tags=test_case.tags or [],
+                )
+        except Exception:
+            pass
```

Ruff (0.15.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)